docs/install.md

```diff
@@ -1,9 +1,22 @@
 # Install guide
 
-You can install Presidio as a service in [Kubernetes](https://kubernetes.io/) or use it as a framework
+You can install Presidio locally using [Docker](), as a service in [Kubernetes](https://kubernetes.io/) or use it as a framework in a python application.
 
 ## The easy way with Docker
 
+You will need to have Docker installed and running, and [make](https://www.gnu.org/software/make/) make installed on your system.
+
+Sync this repo use `make` to build and deploy locally.
```

Comment:
Is this necessary given the new build.sh file?

Reply:
Maybe not, I'm happy to remove it you are.

Reply:
updated now.